!SESSION 2016-12-09 10:20:58.072 -----------------------------------------------
eclipse.buildId=3.7.2.201511261048-RELEASE-e45
java.version=1.8.0_101
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=en_US
Framework arguments:  -product org.springsource.sts.ide
Command-line arguments:  -os win32 -ws win32 -arch x86_64 -product org.springsource.sts.ide

!ENTRY org.eclipse.egit.ui 2 0 2016-12-09 10:24:28.682
!MESSAGE Warning: The environment variable HOME is not set. The following directory will be used to store the Git
user global configuration and to define the default location to store repositories: 'C:\Users\paramanandd.CYBAGE'. If this is
not correct please set the HOME environment variable and restart Eclipse. Otherwise Git for Windows and
EGit might behave differently since they see different configuration options.
This warning can be switched off on the Team > Git > Confirmations and Warnings preference page.

!ENTRY org.eclipse.equinox.p2.transport.ecf 2 0 2016-12-09 10:24:45.796
!MESSAGE Connection to https://spring.io/blog.atom failed on sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target. Retry attempt 0 started
!STACK 0
javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.ssl.Alerts.getSSLException(Unknown Source)
	at sun.security.ssl.SSLSocketImpl.fatal(Unknown Source)
	at sun.security.ssl.Handshaker.fatalSE(Unknown Source)
	at sun.security.ssl.Handshaker.fatalSE(Unknown Source)
	at sun.security.ssl.ClientHandshaker.serverCertificate(Unknown Source)
	at sun.security.ssl.ClientHandshaker.processMessage(Unknown Source)
	at sun.security.ssl.Handshaker.processLoop(Unknown Source)
	at sun.security.ssl.Handshaker.process_record(Unknown Source)
	at sun.security.ssl.SSLSocketImpl.readRecord(Unknown Source)
	at sun.security.ssl.SSLSocketImpl.performInitialHandshake(Unknown Source)
	at sun.security.ssl.SSLSocketImpl.writeRecord(Unknown Source)
	at sun.security.ssl.AppOutputStream.write(Unknown Source)
	at org.apache.http.impl.io.AbstractSessionOutputBuffer.flushBuffer(AbstractSessionOutputBuffer.java:159)
	at org.apache.http.impl.io.AbstractSessionOutputBuffer.flush(AbstractSessionOutputBuffer.java:166)
	at org.apache.http.impl.AbstractHttpClientConnection.doFlush(AbstractHttpClientConnection.java:272)
	at org.apache.http.impl.AbstractHttpClientConnection.flush(AbstractHttpClientConnection.java:277)
	at org.apache.http.impl.conn.AbstractClientConnAdapter.flush(AbstractClientConnAdapter.java:201)
	at org.apache.http.protocol.HttpRequestExecutor.doSendRequest(HttpRequestExecutor.java:239)
	at org.apache.http.protocol.HttpRequestExecutor.execute(HttpRequestExecutor.java:121)
	at org.apache.http.impl.client.DefaultRequestDirector.tryExecute(DefaultRequestDirector.java:685)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:487)
	at org.apache.http.impl.client.AbstractHttpClient.doExecute(AbstractHttpClient.java:863)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:82)
	at org.eclipse.ecf.provider.filetransfer.httpclient4.HttpClientRetrieveFileTransfer.performConnect(HttpClientRetrieveFileTransfer.java:1077)
	at org.eclipse.ecf.provider.filetransfer.httpclient4.HttpClientRetrieveFileTransfer.access$0(HttpClientRetrieveFileTransfer.java:1068)
	at org.eclipse.ecf.provider.filetransfer.httpclient4.HttpClientRetrieveFileTransfer$1.performFileTransfer(HttpClientRetrieveFileTransfer.java:1064)
	at org.eclipse.ecf.filetransfer.FileTransferJob.run(FileTransferJob.java:73)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.validator.PKIXValidator.doBuild(Unknown Source)
	at sun.security.validator.PKIXValidator.engineValidate(Unknown Source)
	at sun.security.validator.Validator.validate(Unknown Source)
	at sun.security.ssl.X509TrustManagerImpl.validate(Unknown Source)
	at sun.security.ssl.X509TrustManagerImpl.checkTrusted(Unknown Source)
	at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(Unknown Source)
	... 24 more
Caused by: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
	at sun.security.provider.certpath.SunCertPathBuilder.build(Unknown Source)
	at sun.security.provider.certpath.SunCertPathBuilder.engineBuild(Unknown Source)
	at java.security.cert.CertPathBuilder.build(Unknown Source)
	... 30 more

!ENTRY com.vwmare.vfabric.ide.eclipse.tcserver.core 4 0 2016-12-09 10:24:51.450
!MESSAGE Error creating server instance with command:
 "D:\Softwares\sts-bundle\pivotal-tc-server-developer-3.1.2.RELEASE\tcruntime-instance.bat create base-instance -t base --force -v 8.0.26.B.RELEASE". Check access permission for the directory D:/Softwares/sts-bundle/pivotal-tc-server-developer-3.1.2.RELEASE and its files and subdirectories.
!SUBENTRY 1 com.vwmare.vfabric.ide.eclipse.tcserver.core 4 0 2016-12-09 10:24:51.450
!MESSAGE The instance creation command resulted in an exception
!STACK 0
java.lang.RuntimeException: java.io.IOException: Cannot run program ""D:\Softwares\sts-bundle\pivotal-tc-server-developer-3.1.2.RELEASE\tcruntime-instance.bat"" (in directory "D:\Softwares\sts-bundle\pivotal-tc-server-developer-3.1.2.RELEASE"): CreateProcess error=2, The system cannot find the file specified
	at com.vmware.vfabric.ide.eclipse.tcserver.internal.core.ServerInstanceCommand.execute(ServerInstanceCommand.java:82)
	at com.vmware.vfabric.ide.eclipse.tcserver.internal.core.TcServerUtil.executeInstanceCreation(TcServerUtil.java:102)
	at com.vmware.vfabric.ide.eclipse.tcserver.internal.core.TcServer21ServerHandlerCallback.configureServer(TcServer21ServerHandlerCallback.java:48)
	at org.springsource.ide.eclipse.commons.internal.configurator.server.ServerDescriptor$ServerHandlerCallbackDelegate.configureServer(ServerDescriptor.java:40)
	at org.springsource.ide.eclipse.commons.configurator.ServerHandler.createServer(ServerHandler.java:318)
	at org.springsource.ide.eclipse.commons.configurator.ServerHandler.createServer(ServerHandler.java:157)
	at org.springsource.ide.eclipse.commons.internal.configurator.server.ConfigurableServerExtension.configure(ConfigurableServerExtension.java:70)
	at org.springsource.ide.eclipse.commons.internal.configurator.ConfiguratorImporter$4.run(ConfiguratorImporter.java:439)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: java.io.IOException: Cannot run program ""D:\Softwares\sts-bundle\pivotal-tc-server-developer-3.1.2.RELEASE\tcruntime-instance.bat"" (in directory "D:\Softwares\sts-bundle\pivotal-tc-server-developer-3.1.2.RELEASE"): CreateProcess error=2, The system cannot find the file specified
	at java.lang.ProcessBuilder.start(Unknown Source)
	at java.lang.Runtime.exec(Unknown Source)
	at org.springsource.ide.eclipse.commons.core.process.StandardProcessRunner.run(StandardProcessRunner.java:58)
	at com.vmware.vfabric.ide.eclipse.tcserver.internal.core.ServerInstanceCommand.execute(ServerInstanceCommand.java:78)
	... 8 more
Caused by: java.io.IOException: CreateProcess error=2, The system cannot find the file specified
	at java.lang.ProcessImpl.create(Native Method)
	at java.lang.ProcessImpl.<init>(Unknown Source)
	at java.lang.ProcessImpl.start(Unknown Source)
	... 12 more
!SUBENTRY 1 com.vwmare.vfabric.ide.eclipse.tcserver.core 4 0 2016-12-09 10:24:51.450
!MESSAGE Output of the instance creation command:


!ENTRY org.springsource.ide.eclipse.commons.configurator 4 0 2016-12-09 10:24:51.452
!MESSAGE The server could not be created.
!STACK 1
org.eclipse.core.runtime.CoreException: Error creating server instance with command:
 "D:\Softwares\sts-bundle\pivotal-tc-server-developer-3.1.2.RELEASE\tcruntime-instance.bat create base-instance -t base --force -v 8.0.26.B.RELEASE". Check access permission for the directory D:/Softwares/sts-bundle/pivotal-tc-server-developer-3.1.2.RELEASE and its files and subdirectories.
	at com.vmware.vfabric.ide.eclipse.tcserver.internal.core.TcServerUtil.handleResult(TcServerUtil.java:157)
	at com.vmware.vfabric.ide.eclipse.tcserver.internal.core.TcServerUtil.executeInstanceCreation(TcServerUtil.java:105)
	at com.vmware.vfabric.ide.eclipse.tcserver.internal.core.TcServer21ServerHandlerCallback.configureServer(TcServer21ServerHandlerCallback.java:48)
	at org.springsource.ide.eclipse.commons.internal.configurator.server.ServerDescriptor$ServerHandlerCallbackDelegate.configureServer(ServerDescriptor.java:40)
	at org.springsource.ide.eclipse.commons.configurator.ServerHandler.createServer(ServerHandler.java:318)
	at org.springsource.ide.eclipse.commons.configurator.ServerHandler.createServer(ServerHandler.java:157)
	at org.springsource.ide.eclipse.commons.internal.configurator.server.ConfigurableServerExtension.configure(ConfigurableServerExtension.java:70)
	at org.springsource.ide.eclipse.commons.internal.configurator.ConfiguratorImporter$4.run(ConfiguratorImporter.java:439)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Contains: The instance creation command resulted in an exception
java.lang.RuntimeException: java.io.IOException: Cannot run program ""D:\Softwares\sts-bundle\pivotal-tc-server-developer-3.1.2.RELEASE\tcruntime-instance.bat"" (in directory "D:\Softwares\sts-bundle\pivotal-tc-server-developer-3.1.2.RELEASE"): CreateProcess error=2, The system cannot find the file specified
	at com.vmware.vfabric.ide.eclipse.tcserver.internal.core.ServerInstanceCommand.execute(ServerInstanceCommand.java:82)
	at com.vmware.vfabric.ide.eclipse.tcserver.internal.core.TcServerUtil.executeInstanceCreation(TcServerUtil.java:102)
	at com.vmware.vfabric.ide.eclipse.tcserver.internal.core.TcServer21ServerHandlerCallback.configureServer(TcServer21ServerHandlerCallback.java:48)
	at org.springsource.ide.eclipse.commons.internal.configurator.server.ServerDescriptor$ServerHandlerCallbackDelegate.configureServer(ServerDescriptor.java:40)
	at org.springsource.ide.eclipse.commons.configurator.ServerHandler.createServer(ServerHandler.java:318)
	at org.springsource.ide.eclipse.commons.configurator.ServerHandler.createServer(ServerHandler.java:157)
	at org.springsource.ide.eclipse.commons.internal.configurator.server.ConfigurableServerExtension.configure(ConfigurableServerExtension.java:70)
	at org.springsource.ide.eclipse.commons.internal.configurator.ConfiguratorImporter$4.run(ConfiguratorImporter.java:439)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: java.io.IOException: Cannot run program ""D:\Softwares\sts-bundle\pivotal-tc-server-developer-3.1.2.RELEASE\tcruntime-instance.bat"" (in directory "D:\Softwares\sts-bundle\pivotal-tc-server-developer-3.1.2.RELEASE"): CreateProcess error=2, The system cannot find the file specified
	at java.lang.ProcessBuilder.start(Unknown Source)
	at java.lang.Runtime.exec(Unknown Source)
	at org.springsource.ide.eclipse.commons.core.process.StandardProcessRunner.run(StandardProcessRunner.java:58)
	at com.vmware.vfabric.ide.eclipse.tcserver.internal.core.ServerInstanceCommand.execute(ServerInstanceCommand.java:78)
	... 8 more
Caused by: java.io.IOException: CreateProcess error=2, The system cannot find the file specified
	at java.lang.ProcessImpl.create(Native Method)
	at java.lang.ProcessImpl.<init>(Unknown Source)
	at java.lang.ProcessImpl.start(Unknown Source)
	... 12 more
Contains: Output of the instance creation command:

!SUBENTRY 1 com.vwmare.vfabric.ide.eclipse.tcserver.core 4 0 2016-12-09 10:24:51.453
!MESSAGE Error creating server instance with command:
 "D:\Softwares\sts-bundle\pivotal-tc-server-developer-3.1.2.RELEASE\tcruntime-instance.bat create base-instance -t base --force -v 8.0.26.B.RELEASE". Check access permission for the directory D:/Softwares/sts-bundle/pivotal-tc-server-developer-3.1.2.RELEASE and its files and subdirectories.
!SUBENTRY 2 com.vwmare.vfabric.ide.eclipse.tcserver.core 4 0 2016-12-09 10:24:51.453
!MESSAGE The instance creation command resulted in an exception
!STACK 0
java.lang.RuntimeException: java.io.IOException: Cannot run program ""D:\Softwares\sts-bundle\pivotal-tc-server-developer-3.1.2.RELEASE\tcruntime-instance.bat"" (in directory "D:\Softwares\sts-bundle\pivotal-tc-server-developer-3.1.2.RELEASE"): CreateProcess error=2, The system cannot find the file specified
	at com.vmware.vfabric.ide.eclipse.tcserver.internal.core.ServerInstanceCommand.execute(ServerInstanceCommand.java:82)
	at com.vmware.vfabric.ide.eclipse.tcserver.internal.core.TcServerUtil.executeInstanceCreation(TcServerUtil.java:102)
	at com.vmware.vfabric.ide.eclipse.tcserver.internal.core.TcServer21ServerHandlerCallback.configureServer(TcServer21ServerHandlerCallback.java:48)
	at org.springsource.ide.eclipse.commons.internal.configurator.server.ServerDescriptor$ServerHandlerCallbackDelegate.configureServer(ServerDescriptor.java:40)
	at org.springsource.ide.eclipse.commons.configurator.ServerHandler.createServer(ServerHandler.java:318)
	at org.springsource.ide.eclipse.commons.configurator.ServerHandler.createServer(ServerHandler.java:157)
	at org.springsource.ide.eclipse.commons.internal.configurator.server.ConfigurableServerExtension.configure(ConfigurableServerExtension.java:70)
	at org.springsource.ide.eclipse.commons.internal.configurator.ConfiguratorImporter$4.run(ConfiguratorImporter.java:439)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: java.io.IOException: Cannot run program ""D:\Softwares\sts-bundle\pivotal-tc-server-developer-3.1.2.RELEASE\tcruntime-instance.bat"" (in directory "D:\Softwares\sts-bundle\pivotal-tc-server-developer-3.1.2.RELEASE"): CreateProcess error=2, The system cannot find the file specified
	at java.lang.ProcessBuilder.start(Unknown Source)
	at java.lang.Runtime.exec(Unknown Source)
	at org.springsource.ide.eclipse.commons.core.process.StandardProcessRunner.run(StandardProcessRunner.java:58)
	at com.vmware.vfabric.ide.eclipse.tcserver.internal.core.ServerInstanceCommand.execute(ServerInstanceCommand.java:78)
	... 8 more
Caused by: java.io.IOException: CreateProcess error=2, The system cannot find the file specified
	at java.lang.ProcessImpl.create(Native Method)
	at java.lang.ProcessImpl.<init>(Unknown Source)
	at java.lang.ProcessImpl.start(Unknown Source)
	... 12 more
!SUBENTRY 2 com.vwmare.vfabric.ide.eclipse.tcserver.core 4 0 2016-12-09 10:24:51.453
!MESSAGE Output of the instance creation command:

